src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertToJsonCommand.cs

```diff
@@ -15,6 +15,7 @@ namespace Microsoft.PowerShell.Commands
     /// The ConvertTo-Json command.
```

Comment:
@iSazonov, your last commit had 47 failures in `PowerShell-CI-macos`
(These are 5 of the failures)

Error occurred in test script '/Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1'
```powershell
At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:288 char:22
+             ${psObj}?.name | Should -BeExactly 'value'
+                      ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:288 char:22
+             ${psObj}?.name | Should -BeExactly 'value'
+                      ~~~~~
Unexpected token '.name' in expression or statement.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:289 char:22
+             ${array}?.length | Should -Be 3
+                      ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:289 char:22
+             ${array}?.length | Should -Be 3
+                      ~~~~~~~
Unexpected token '.length' in expression or statement.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:290 char:21
+             ${hash}?.a | Should -Be 1
+                     ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:290 char:21
+             ${hash}?.a | Should -Be 1
+                     ~~
Unexpected token '.a' in expression or statement.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:292 char:36
+             (Get-Process -Id $pid)?.Name | Should -BeLike "pwsh*"
+                                    ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:292 char:36
+             (Get-Process -Id $pid)?.Name | Should -BeLike "pwsh*"
+                                    ~~~~~
Unexpected token '.Name' in expression or statement.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:293 char:35
+             (Get-Item $TestDrive)?.EnumerateFiles()?.Name | Should -B ?
+                                   ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:293 char:35
+             (Get-Item $TestDrive)?.EnumerateFiles()?.Name | Should -B ?
+                                   ~~~~~~~~~~~~~~~
Unexpected token '.EnumerateFiles' in expression or statement.

Not all parse errors were reported.  Correct the reported errors and try again.
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1'
```powershell
At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:288 char:22
+             ${psObj}?.name | Should -BeExactly 'value'
+                      ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:288 char:22
+             ${psObj}?.name | Should -BeExactly 'value'
+                      ~~~~~
Unexpected token '.name' in expression or statement.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:289 char:22
+             ${array}?.length | Should -Be 3
+                      ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:289 char:22
+             ${array}?.length | Should -Be 3
+                      ~~~~~~~
Unexpected token '.length' in expression or statement.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:290 char:21
+             ${hash}?.a | Should -Be 1
+                     ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:290 char:21
+             ${hash}?.a | Should -Be 1
+                     ~~
Unexpected token '.a' in expression or statement.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:292 char:36
+             (Get-Process -Id $pid)?.Name | Should -BeLike "pwsh*"
+                                    ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:292 char:36
+             (Get-Process -Id $pid)?.Name | Should -BeLike "pwsh*"
+                                    ~~~~~
Unexpected token '.Name' in expression or statement.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:293 char:35
+             (Get-Item $TestDrive)?.EnumerateFiles()?.Name | Should -B ?
+                                   ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:293 char:35
+             (Get-Item $TestDrive)?.EnumerateFiles()?.Name | Should -B ?
+                                   ~~~~~~~~~~~~~~~
Unexpected token '.EnumerateFiles' in expression or statement.

Not all parse errors were reported.  Correct the reported errors and try again.
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1'
```powershell
At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:288 char:22
+             ${psObj}?.name | Should -BeExactly 'value'
+                      ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:288 char:22
+             ${psObj}?.name | Should -BeExactly 'value'
+                      ~~~~~
Unexpected token '.name' in expression or statement.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:289 char:22
+             ${array}?.length | Should -Be 3
+                      ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:289 char:22
+             ${array}?.length | Should -Be 3
+                      ~~~~~~~
Unexpected token '.length' in expression or statement.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:290 char:21
+             ${hash}?.a | Should -Be 1
+                     ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:290 char:21
+             ${hash}?.a | Should -Be 1
+                     ~~
Unexpected token '.a' in expression or statement.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:292 char:36
+             (Get-Process -Id $pid)?.Name | Should -BeLike "pwsh*"
+                                    ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:292 char:36
+             (Get-Process -Id $pid)?.Name | Should -BeLike "pwsh*"
+                                    ~~~~~
Unexpected token '.Name' in expression or statement.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:293 char:35
+             (Get-Item $TestDrive)?.EnumerateFiles()?.Name | Should -B ?
+                                   ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:293 char:35
+             (Get-Item $TestDrive)?.EnumerateFiles()?.Name | Should -B ?
+                                   ~~~~~~~~~~~~~~~
Unexpected token '.EnumerateFiles' in expression or statement.

Not all parse errors were reported.  Correct the reported errors and try again.
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Built-in type information tests.Should have correct number of built-in type items in type table
```powershell
Expected exactly 272, but got 271.
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/s/test/powershell/engine/ETS/TypeTable.Tests.ps1: line 21
21:         $types.Count | Should -BeExactly ($IsWindows ? 273 : 272)
```
TabCompletion.Should complete dotnet method with null conditional operator
```powershell
Expected exactly 'ToString(', but got $null.
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/s/test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1: line 38
38:         $res.CompletionMatches[0].CompletionText | Should -BeExactly 'ToString('
```